src/main/java/hw_11/Task3.java

```diff
@@ -0,0 +1,32 @@
+package hw_11;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class Task3 {
+    public static void main(String[] args) {
+        Queue<String> queue = new LinkedList<>();
+        queue.offer("English");
+        queue.offer("German");
+        queue.offer("Ukrainian");
+        queue.offer("Polish");
+        queue.offer("French");
+        System.out.println(queue);
+
+        System.out.println(queue.size());
+        System.out.println(queue.peek());
+        System.out.println(queue);
+        System.out.println(queue.poll());
+        System.out.println(queue);
+        System.out.println(queue.remove());
+        System.out.println(queue);
+        System.out.println(queue.isEmpty());
+        System.out.println(queue);
+
+        Queue<String> newqueue = new LinkedList<>();
+        System.out.println(newqueue.element());
```

Comment:
When I run this task#3 then I get an error "NoSuchElementException". Could you fix it please?
![Screenshot_5](https://github.com/user-attachments/assets/9f1125c3-47a1-4101-a516-b45bdc70ad63)

Reply:
fixed